### PR TITLE
Allow resizing 'Text' Column instead 'Gap to next in seconds'

### PR DESCRIPTION
**Pull request by ivandrofly**
This is really useful, most of the form are using SE's list view this patch will make SE Listview more flexible

### CODE REVIEW

File: src/Controls/SubtitleListView.cs [hunk 1]
```diff
@@ -1104,5 +1104,12 @@ public void ShowAllColumns()
             }
         }
 
+        public void SetCustomResize(EventHandler handler)
+        {
+            if (handler == null)
+                return;
+            Resize -= SubtitleListViewResize;
+            Resize += handler;
```
Comment (ivandrofly) [inline src/Controls/SubtitleListView.cs]: Remove default re-size method and add our new one

**Comment by ivandrofly** [comment]
Any thoughts /cc @niksedk

**Comment by niksedk** [comment]
Looks good :)

**Comment by ivandrofly** [comment]
:wink:  Seems like we are having merge conflict, let me rebase first

**Comment by niksedk** [comment]
super

**Comment by ivandrofly** [comment]
Done!

**Pull request merged**